tests/dummy/app/router.js

```diff
@@ -10,6 +10,7 @@ Router.map(function() {
   this.route('moment-examples');
   this.route('clear-button-examples');
   this.route('range-examples');
+  this.route('custom-actions-examples');
```

Comment:
"custom-actions" it's not the best name. If this commit is adding support for a "custom trigger", then this should be "custom-trigger-examples".